kong/plugins/zipkin/reporter.lua

```diff
@@ -25,6 +25,17 @@ local span_kind_map = {
 	producer = "PRODUCER";
 	consumer = "CONSUMER";
 }
+
+-- Utility function to get zipkin tags from opentracing span.
```

Comment:
I don't it's worth having this as it's own function (as it only has a single call-site).
Could you inline it into the `:report` method?

Reply:
It's Ok 👍